app/serializers/framework_assessment_serializer.rb

```diff
@@ -24,7 +24,7 @@ class FrameworkAssessmentSerializer
   attribute :editable, &:editable?
 
   meta do |object|
-    { section_progress: object.section_progress }
+    { section_progress: object.calculate_section_progress }
```

Comment:
I'm a bit confused by this - isn't the calculation already persisted in the new `section_progress` attribute of the framework, so it can just be serialized directly as an attribute?

Reply:
...unless you're planning to make this change in a future PR once all the data has been backfilled?

Reply:
As expected, you're waaaaay ahead of me and this is covered in https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/1281 😁